app/main.py

```diff
@@ -1 +1,55 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.appetite <= 0 or not self.is_hungry:
+            return 0
+        print(f"Eating {self.appetite} food points...")
+        eat_food = self.appetite
+        self.appetite -= self.appetite
```

Comment:
no need to do this, u should just return an appetite points, no need to change them